src/com/theironyard/novauc/MarvelComics.java

```diff
@@ -0,0 +1,55 @@
+package com.theironyard.novauc;
+
+/**
+ * Created by vtcurry on 2/7/17.
+ */
+public class MarvelComics {
+    String CompanyName;
```

Comment:
Please note Naming Convention for Java variables is  that first letter be lower case.  e.g. CompanyName would be named companyName,  ref: 
http://docs.oracle.com/javase/tutorial/java/nutsandbolts/variables.html.     see Naming section.

This is a non-issue for program functionality.